geode-core/src/main/java/org/apache/geode/management/internal/configuration/callbacks/ConfigurationChangeListener.java

```diff
@@ -95,6 +89,14 @@ private void addOrRemoveJarFromFilesystem(EntryEvent<String, Configuration> even
         }
       }
     }
+
+    for (String jarAdded : newJars) {
+      try {
+        sharedConfig.downloadJarFromOtherLocators(group, jarAdded);
+      } catch (Exception e) {
+        logger.error("Unable to add jar: " + jarAdded, e);
```

Comment:
Similarly, is it safe for this and/or the exception above to be swallowed?  Particularly if you can't add the jar, I imagine this should rethrow.  Or is that something that would be addressed as a matter of course as enforced by the cluster config?